### PR TITLE
Initial details for connector devops and integration

### PR DESCRIPTION
**Pull request by snehahegde7**
Initial set of changes for connector devops process, integration and deployment with cp4waiops.

### CODE REVIEW

File: Connector-devops-cicd-process.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+```
+#### Connector templates based on grpc-java-sdk
+Release versions of the sdk can be found [here](https://github.ibm.com/quicksilver/grpc-java-sdk/releases). Make sure
+that `connector/pom.xml` uses the correct version. For example, `v1.0.1` maps to `1.0.1` in the `pom.xml` file.
+
+To upgrade to another version of the SDK, modify the value of `CONNECTOR_SDK_VERSION` in the `Makefile`.
+
+1. Set the environment variable GITHUB_TOKEN to a github api key with full repo access.
+2. Run `make download-connector-sdk` from the `open-liberty` folder.
+- Example template :https://github.com/IBM/cp4waiops-connectors-java-template
```
Comment (Kyle-Cooley) [inline Connector-devops-cicd-process.md:31]: This process will change. Let's note we need to update this documentation once the maven artifact is available.

File: Connector-devops-cicd-process.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+### CICD automation
+- Configure travis or jenkins job for the connector as part of CICD process to automatically build and push the connector image to artifactory location.
+- The image can be used for deploying on cp4waiops cluster by creating BundleManifest artifacts.
+
+### k8s deployment files in bundle manifest format
+- Create the prerequisites artifacts :
+   - Connectorschema : For creating a connector for a data source that is not yet supported in Watson AIOps you first need to create a ConnectorSchema. ConnectorSchema CR provides content for the UI to render the corresponding form in the Data and tool integrations section of the product and it validates completed UI forms. Refer the example in ui-schema details section.
+   - Microedgeconfiguration CR: If the connector supports remote orchestration create the microedgeconfiguration CR in order for the bootstrap script to provide the container image location to download and run on a remote VM.
+- Create the standard k8s deployment artifacts : Deployment, service, serviceaccount etc.
+- Refer example BundleManifest template with folder structre and files : (add link when available)
```
Comment (Kyle-Cooley) [inline Connector-devops-cicd-process.md:42]: If you need a link, the sample repo has this. The bundle:
https://github.com/IBM/cp4waiops-connectors-java-template/blob/main/bundlemanifest.yaml
The deployment artifacts:
https://github.com/IBM/cp4waiops-connectors-java-template/tree/main/bundle-artifacts

**Review by Kyle-Cooley — APPROVED** [review]

File: Connector-devops-cicd-process.md (hunk 1 above)
Comment (snehahegde7) [reply Connector-devops-cicd-process.md:31]: Sure 👍

File: Connector-devops-cicd-process.md (hunk 2 above)
Comment (snehahegde7) [reply Connector-devops-cicd-process.md]: done

**Pull request merged**